### PR TITLE
Add option to disable owner email address in CSR subject

### PR DESCRIPTION
**Pull request by intgr**

### CODE REVIEW

File: lemur/certificates/service.py [hunk 1]
```diff
@@ -344,8 +344,9 @@ def create_csr(**csr_config):
     private_key = generate_private_key(csr_config.get('key_type'))
 
     builder = x509.CertificateSigningRequestBuilder()
-    name_list = [x509.NameAttribute(x509.OID_COMMON_NAME, csr_config['common_name']),
-                 x509.NameAttribute(x509.OID_EMAIL_ADDRESS, csr_config['owner'])]
+    name_list = [x509.NameAttribute(x509.OID_COMMON_NAME, csr_config['common_name'])]
+    if current_app.config.get('LEMUR_OWNER_EMAIL_IN_SUBJECT', True):
```
Comment (intgr) [inline lemur/certificates/service.py]: Do you think this should even default to `True`?

Comment (kevgliss) [reply lemur/certificates/service.py]: Have you encountered CAs that don't allow this? Or is this purely from a privacy consideration?

Comment (intgr) [reply lemur/certificates/service.py]: It's for privacy, yes.

Comment (kevgliss) [reply lemur/certificates/service.py]: I would then leave the default as `True` and let them opt out of it. So as not to break the existing functionality.

Comment (intgr) [reply lemur/certificates/service.py]: Sure, works for me

**Comment by kevgliss** [comment]
lgtm thanks!

**Pull request merged**